draft-ietf-asdf-sdf-protocol-mapping.md

```diff
@@ -550,8 +550,14 @@ hema/HeartRate/properties/pulse"
 While SDF provides a way to describe a device, a method is needed to associate a
 mapping between an instance of a device and its associated SDF models. To
 accomplish this, we define a SCIM extension that can be used in conjunction with
-{{?I-D.ietf-scim-device-model}}, the details of which can be found in
-{{scim-sdf-extension-schema}}.
+{{!I-D.ietf-scim-device-model}} in {{scim-sdf-extension-schema}}. Implementation
+of this SCIM extension is OPTIONAL and independent of the protocol mapping
+functionality defined in the rest of this document.
+
```

Comment:
Adding a reference to the place where the attributes are defined is useful here.

```suggestion
The SCIM schema attributes used here are described in Section 7 of {{!RFC7643}}.

```